q2_alignment/plugin_setup.py

```diff
@@ -40,6 +42,37 @@
     citations=[citations['katoh2013mafft']]
 )
 
+plugin.methods.register_function(
+    function=q2_alignment.sina,
+    inputs={'sequences': FeatureData[Sequence],
+            'reference': FeatureData[AlignedSequence]},
+    parameters={'arb_reference': Str,
+                'num_references': Int % Range(0, None),
+                'kmer_len': Int % Range(5, 25)},
+    outputs=[('alignment', FeatureData[AlignedSequence])],
+    input_descriptions={
+        'sequences': 'The sequences to be aligned.',
+        'reference': 'The reference alignment (QZA format).',
+    },
+    parameter_descriptions={
+        'arb_reference': 'The reference alignment (ARB format).',
+        'num_references': 'The number of reference sequences used to '
+                          'align each sequence.',
+        'kmer_len': 'The size of K used in reference search. '
+                    'Use 6-8 for sequences <150bp. '
+                    'Use 10 for full length 16S. '},
+    output_descriptions={'alignment': 'The aligned sequences.'},
+    name='Reference based multiple sequence alignment with SINA',
+    description=dedent("""
+    Perform reference based multiple sequence alignment using SINA.
+
+    Note:
```

Comment:
Could you drop this note here (which I think would also let you drop the `dedent` import and call, in favor of making the `input_description` for `reference` say something like the following:

'The reference alignment (QZA format) - an ARB database can alternatively be provided using the `arb_reference` parameter.'

With some interfaces, this description might not show up in a place that will be intuitive for this type of help information. For example, in a GUI, it might not even show up in the same view as where the user is working with these parameters.

Reply:
fixed in 92ad05a